cmd/metrics-v2.go

```diff
@@ -49,6 +50,7 @@ func init() {
 		getNodeHealthMetrics(),
 		getClusterStorageMetrics(),
 		getClusterTierMetrics(),
+		getKMSMetrics(),
```

Comment:
Not fully sure - but I think this needs to be added to peerMetricsGroups to show KMS errors of all nodes.

@aead did you test with multiple nodes setup ?

Reply:
@vadmeste the metric information is obtained directly from KES instance by calling the client API - there is no reason to capture this from all nodes AFAICS.

Reply:
Added a separate node metric whether the KMS is online/offline. The remaining metrics are cluster metrics.